### ISSUE TITLE
Add Content to EDD Errors

### ISSUE THREAD

**Issue by chriscct7** [easydigitaldownloads/Easy-Digital-Downloads#3242]
we should add support for contexts in EDD errors
- "checkout"
- "customer"
etc

**Comment by pippinsplugins** [comment]
To elaborate a little, this could help us resolve issues where errors for one screen accidentally pollute another screen.

For example, if an error is logged somewhere unrelated while updating a customer, the customer update will detect that error and fail.

**Comment by cklosowski** [comment]
Are we thinking maybe when you register your error with EDD, you add a context, then when doing `edd_has_errors( $context )` can only return the errors if they meet context?

**Comment by chriscct7** [comment]
Yeah, and maybe either a view or action parameter (or both) would be nice.

**Comment by pippinsplugins** [comment]
Yep! Rcp does this via a third parameter in the add error function.

**Comment by pippinsplugins** [comment]
Punting to 2.6.
